docs/make_a_release.rst

```diff
@@ -93,98 +92,55 @@ Setting up environment
       expression: ``^[0-9]+(\.[0-9]+)*(\.post[0-9]+)?$``.
 
 
-5. In `README.rst`, update `PyPI`_ download count after running `this big table query <https://console.cloud.google.com/bigquery?sq=280188050539:6571a5b49fd1426395e4beea055d2b1b>`_
-   and commit the changes.
-
-  .. code::
-
-    $ git add README.rst && \
-      git commit -m "README: Update download stats [ci skip]"
-
-  ..  note::
-
-    To learn more about `pypi-stats`, see `How to get PyPI download statistics <https://kirankoduru.github.io/python/pypi-stats.html>`_.
-
-
-6. In `CHANGES.rst` replace ``Next Release`` section header with
+5. In `CHANGES.rst` replace ``Next Release`` section header with
    ``Scikit-build X.Y.Z`` and commit the changes.
 
   .. code::
 
     $ git add CHANGES.rst && \
-      git commit -m "Scikit-build ${release}"
+      git commit -m "Scikit-build $release"
 
 
-7. Tag the release
+6. Tag the release
 
   .. code::
 
-    $ git tag --sign -m "Scikit-build ${release}" ${release} master
+    $ git tag --sign -m "Scikit-build $release" $release master
 
   .. warning::
 
       We recommend using a `GPG signing key <https://help.github.com/articles/generating-a-new-gpg-key/>`_
       to sign the tag.
 
 
-8. Create the source distribution and wheel
+7. Publish the both release tag and the master branch
 
   .. code::
 
-    $ python setup.py sdist bdist_wheel
-
-
-9. Publish the both release tag and the master branch
-
-  .. code::
-
-    $ git push origin ${release} && \
+    $ git push origin $release && \
       git push origin master
 
 
-10. Upload the distributions on `PyPI`_
-
-  .. code::
-
-    twine upload dist/*
-
-  .. note::
+..
+    8. Make a `GitHub release <https://github.com/scikit-build/scikit-build/releases/new>`_. Paste the converted release notes as markdown; convert using
```

Comment:
This commented out section is how I'd normally prepare release notes. Ever since GitHub started allowing users to watch releases on repos (about a year ago?), it's nice to get the release notes directly in the notification, as well as dependabot picks it up if you use it. @jcfr, would you be okay to transition to including the release notes in the GitHub release?

Reply:
> would you be okay to transition to including the release notes in the GitHub release?

That is sensible. 

In the past, to convert to markdown, I had luck copy/pasting the html in a draft [discourse](https://discourse.slicer.org/) post. 
Looking for online converter ... I also found  [clipboard2markdown](https://euangoddard.github.io/clipboard2markdown/). See below.

To simplify the process, look like using `clipboard2markdown` may work.



| using  [clipboard2markdown](https://euangoddard.github.io/clipboard2markdown/) | using [discourse](https://discourse.slicer.org/) |
|--|---|
| ![image](https://user-images.githubusercontent.com/219043/128819392-7efa34cc-d9ed-41d7-99c7-39bdc5844124.png) ![image](https://user-images.githubusercontent.com/219043/128819316-91d0fdcc-25c6-4791-84fa-d16cebac29d4.png) | ![image](https://user-images.githubusercontent.com/219043/128818908-7dc94407-9e74-4c87-a0b2-17baebef1bd7.png) |